index.html

```diff
@@ -66,21 +76,121 @@
     </nav>
 
     <!-- Page Content -->
-    <div class="container">
-
-        <div class="row">
-            <div class="col-lg-12 text-center">
-                <h1>A Bootstrap Starter Template</h1>
-                <p class="lead">Complete with pre-defined file paths that you won't have to change!</p>
-                <ul class="list-unstyled">
-                    <li>Bootstrap v3.3.6</li>
-                    <li>jQuery v1.11.1</li>
-                </ul>
-            </div>
+    <div class="container" id="Home">
+        <div id="Location">
+            <h4>LOCATION     1914 translation by H. Rackham</h4>
```

Comment:
As placeholders in the sections, please just have the title.
```
<h2>Location</2>
```

Also, you should never capitalize words in html. If you want want to capitalize it, you can use css with ```text-transform: uppercase;```